e2e/backend.e2e.spec.ts

```diff
@@ -110,10 +110,7 @@ test.describe.serial('KubernetesBackend', () => {
         },
       };
       /** Platform-specific changes to `newSettings`. */
-      const platformSettings: Partial<Record<NodeJS.Platform, RecursivePartial<Settings>>> = {
-        win32:  { virtualMachine: { hostResolver: getAlternateSetting(currentSettings, 'virtualMachine.hostResolver', true, false) } },
-        darwin: { experimental: { virtualMachine: { useRosetta: getAlternateSetting(currentSettings, 'experimental.virtualMachine.useRosetta', true, false) } } },
-      };
+      const platformSettings: Partial<Record<NodeJS.Platform, RecursivePartial<Settings>>> = { darwin: { experimental: { virtualMachine: { useRosetta: getAlternateSetting(currentSettings, 'experimental.virtualMachine.useRosetta', true, false) } } } };
```

Comment:
Please pick a new windows-only setting to use, because this is used to test that (on macOS) we will handle unsupported settings correctly.